### ISSUE TITLE
Installation of tfsec Fails from Version 1.28.8 Due to Format Change

### ISSUE THREAD

**Issue by leiicamundi** [woneill/asdf-tfsec#34]
Hello :)

**Describe the bug**

Since tfsec version 1.28.8 (see https://github.com/aquasecurity/tfsec/releases/tag/v1.28.8), the format delivered by goreleaser has changed. Now, the release is provided as a tar.gz archive containing the binaries. As a result, any attempt to install tfsec version 1.28.8 or higher fails.

**Steps to reproduce**

1. Attempt to install tfsec version 1.28.8 or higher using asdf.
2. Observe the failure message during the installation process.

**Expected behavior**

The installation process should correctly handle the tar.gz format and successfully install tfsec version 1.28.8 or higher.

**Screenshots**

```
asdf: Warn: the current plugin (tflint) does not support that. Downloaded files will not be preserved.
* Downloading tfsec release 1.28.9...
curl: (22) The requested URL returned error: 404
asdf-tfsec: Could not download https://github.com/aquasecurity/tfsec/releases/download/v1.28.9/tfsec-linux-amd64
Error: Action failed with error Error: The process '/home/runner/.asdf/bin/asdf' failed with exit code 1
```

**Additional context**

The issue arises due to the format change in the delivery method starting from version 1.28.8. The previous versions used a different format, which is why the installation process fails to locate and download the necessary binaries from the provided URL.

Thank you for maintaining this plugin!

**Comment by woneill** [comment]
Thanks for the bug report!
I'm about to head out on vacation this week so I'll take a look at this next week. I wanted to make sure you had some feedback that your bug report was seen :)

**Comment by woneill** [comment]
@leiicamundi It looks like the tfsec project published some bad releases between v1.28.7 and v1.28.9 while they were working out the goreleaser upgrade. v1.28.10 appears to have published binaries correctly and this plugin is working as expected:

```
> for version in 1.28.6 1.28.7 1.28.8 1.28.9 1.28.10; do
asdf plugin test tfsec . --asdf-tool-version $version "tfsec --version"
done
Location of tfsec plugin: /var/folders/3q/_20wm625725dhysxxby7dlvw0000gn/T/asdf.XXXX.bNaEsI2dij/plugins/tfsec
Updating tfsec to main
Already on 'main'
Your branch is up to date with 'origin/main'.
* Downloading tfsec release 1.28.6...
tfsec 1.28.6 installation was successful!

======================================================
tfsec is joining the Trivy family

tfsec will continue to remain available
for the time being, although our engineering
attention will be directed at Trivy going forward.

You can read more here:
https://github.com/aquasecurity/tfsec/discussions/1994
======================================================
v1.28.6
Location of tfsec plugin: /var/folders/3q/_20wm625725dhysxxby7dlvw0000gn/T/asdf.XXXX.1802CyKdxh/plugins/tfsec
Updating tfsec to main
Already on 'main'
Your branch is up to date with 'origin/main'.
* Downloading tfsec release 1.28.7...
curl: (22) The requested URL returned error: 404
asdf-tfsec: Could not download https://github.com/aquasecurity/tfsec/releases/download/v1.28.7/tfsec-darwin-arm64
FAILED: install exited with an error
Location of tfsec plugin: /var/folders/3q/_20wm625725dhysxxby7dlvw0000gn/T/asdf.XXXX.LdFW07kWx1/plugins/tfsec
Updating tfsec to main
Already on 'main'
Your branch is up to date with 'origin/main'.
* Downloading tfsec release 1.28.8...
curl: (22) The requested URL returned error: 404
asdf-tfsec: Could not download https://github.com/aquasecurity/tfsec/releases/download/v1.28.8/tfsec-darwin-arm64
FAILED: install exited with an error
Location of tfsec plugin: /var/folders/3q/_20wm625725dhysxxby7dlvw0000gn/T/asdf.XXXX.4idFFnl12T/plugins/tfsec
Updating tfsec to main
Already on 'main'
Your branch is up to date with 'origin/main'.
* Downloading tfsec release 1.28.9...
curl: (22) The requested URL returned error: 404
asdf-tfsec: Could not download https://github.com/aquasecurity/tfsec/releases/download/v1.28.9/tfsec-darwin-arm64
FAILED: install exited with an error
Location of tfsec plugin: /var/folders/3q/_20wm625725dhysxxby7dlvw0000gn/T/asdf.XXXX.UyLwK3np0C/plugins/tfsec
Updating tfsec to main
Already on 'main'
Your branch is up to date with 'origin/main'.
* Downloading tfsec release 1.28.10...
tfsec 1.28.10 installation was successful!

======================================================
tfsec is joining the Trivy family

tfsec will continue to remain available
for the time being, although our engineering
attention will be directed at Trivy going forward.

You can read more here:
https://github.com/aquasecurity/tfsec/discussions/1994
======================================================
v1.28.10
```
